source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -61,15 +61,31 @@ let initialize_configuration
           ~maximum_tito_depth
     |> TaintConfiguration.exception_on_error
   in
+  let () =
+    Statistics.performance
+      ~name:"Verified taint configuration"
+      ~phase_name:"Verified taint configuration"
+      ~timer
+      ()
+  in
```

Comment:
Not sure if it is useful to log this, but I guess it also doesn't hurt. cc @arthaud

Reply:
Yeah, not much most of the time it just takes a split of a micro second.